torch/csrc/jit/runtime/static/impl.cpp

```diff
@@ -1042,7 +1042,7 @@ void BlockRunner::set_inputs(IValueList&& args, const KeywordArgs& kwargs) {
     const auto& schema_arg = schema_args[i_arg];
 
     if (i_arg - 1 < args.size()) {
-      check_type(schema_arg, std::forward<IValueList>(args)[i_arg - 1]);
+      check_type(schema_arg, args[i_arg - 1]);
```

Comment:
What error does this resolve?

Reply:
Tidy warns use-after-move. Although that isn't real, removing std::forward can make the code cleaner.